### DISCUSSION TITLE
implement phobos for upbge fork (clone repo) - help wrap BT_MultiBody in upbge/bge fork of blender.

### DISCUSSION THREAD

**Discussion by BluePrintRandom** [dfki-ric/phobos#292] [General]
1. We currently do not have the physics type 'multibody' in upbge - the game engine development fork of blender

Help us wrap BT_Multibody using pieces of the already existing pybullet code to generate multibody agents.

**Additional context**
Currently we can export a URDF from blender to pybullet 
upbge is a game engine layer restored to blender that was removed (BGE) now using eevee and modifiers / bpy in realtime.
having multibody agents could allow for animations to be recorded and then played back later in cycles / rendered offline* 
(so proxy rigs / gameplay can make movies)

 - robot in upbge using ik/ armatures https://www.youtube.com/watch?v=fMwe9hGZb9w
 
 - record gameplay as f-curves - https://blenderartists.org/t/blender-upbge-as-a-animation-tool-recording-keyframes-physics-agents/1235434/7?u=blueprintrandom 
 
[upbge.org/#/](https://upbge.org/#/)

**Comment by hwiedPro** [comment]
We can't support this from our side as we have no funding to do so.
Nevertheless, I find this a good discussion to which the community might contribute.
Therefore I'll convert this issue to a discussion
